geth-dev-node/Dockerfile

```diff
@@ -1,64 +1,19 @@
 FROM ethereum/client-go:latest
-COPY genesis.json /app/genesis.json
-RUN echo "p" > /app/password.txt
-RUN geth makedag 0 /app/.ethash
+
+RUN mkdir /home/\.ethereum
+RUN mkdir /home/\.ethereum/keystore
+RUN mkdir /home/\.ethash
+
+COPY genesis.json /home/genesis.json
+RUN printf "\n" > /home/password.txt
+# Sealing account (pre-funded with a large balance)
+COPY UTC--2017-10-16T19-13-42.464934000Z--c5ed899b0878656feb06467e2e9ede3ae73cbcb7 /home/\.ethereum/keystore/UTC--2017-10-16T19-13-42.464934000Z--c5ed899b0878656feb06467e2e9ede3ae73cbcb7
```

Comment:
Recommend renaming this file to something much simpler like `dev-key.json`.  The file names for keys doesn't have any impact on how they are used by Geth.

Reply:
Yeah, I don't like this naming convention, but I tried renaming them, and for some reason the accounts weren't getting unlocked.  I guess maybe something else was preventing the unlocking.  I'll check on this.